entity-framework/toc.yml

```diff
@@ -411,7 +411,7 @@
         href: core/cli/services.md
 
     - name: EF Core API reference >>
-      href: /dotnet/api/?view=efcore-6.0
+      href: /dotnet/api/?view=efcore-6.0&preserve-view=true
```

Comment:
They recommend not using the version in URL string so it can go to latest by default but since it is a shared docset there is no way for us to specify efcore without using the version (monikers are tied to version). So the later arg explicitly state that we want 6.0 version and don't generate warning.